### ISSUE TITLE
Keyboard and DS4 not working anymore

### ISSUE THREAD

**Issue by Cimmerian-Iter** [xan1242/NFSU-XtendedInput#25]
Hi I'm on windows 8 and tried both normal package and win 7 package and when i try to use my ds4 or keyboard on need for speed underground they both don't work, only mouse does something. I thought it was aproblem with widscreen fix but even disabling it and having 4:3 screen ratio the controls doesn't work. Any clues?

**Comment by xan1242** [comment]
For DS4 to work you need either DS4Windows or Steam to convert it to XInput.

**Comment by Cimmerian-Iter** [comment]
Yes obviously..... I use ds4windows. Still that doesn't explain why even the keyboard doesn't work in game. I would understand if it was only the DS4 not working
